docs/_src/api/api/pipelines.md

```diff
@@ -564,6 +565,67 @@ we cut the context on the same side, recalculate the score and take the mean of
 Thus [AB] <-> [BC] (score ~50) gets recalculated with B <-> B (score ~100) scoring ~75 in total.
 - `context_matching_threshold`: Score threshold that candidates must surpass to be included into the result list. Range: [0,100]
 
+<a id="base.Pipeline.eval_batch"></a>
+
+#### Pipeline.eval\_batch
+
+```python
+@send_event
+def eval_batch(labels: List[MultiLabel], documents: Optional[List[List[Document]]] = None, params: Optional[dict] = None, sas_model_name_or_path: Optional[str] = None, sas_batch_size: int = 32, sas_use_gpu: bool = True, add_isolated_node_eval: bool = False, custom_document_id_field: Optional[str] = None, context_matching_min_length: int = 100, context_matching_boost_split_overlaps: bool = True, context_matching_threshold: float = 65.0) -> EvaluationResult
+```
+
+Evaluates the pipeline by running the pipeline in batches in debug mode
+
+and putting together all data that is needed for evaluation, e.g. calculating metrics.
+
+If you want to calculate SAS (Semantic Answer Similarity) metrics, you have to specify `sas_model_name_or_path`.
+
+You will be able to control the scope within which an answer or a document is considered correct afterwards (See `document_scope` and `answer_scope` params in `EvaluationResult.calculate_metrics()`).
+Some of these scopes require additional information that already needs to be specified during `eval()`:
+- `custom_document_id_field` param to select a custom document ID from document's meta data for ID matching (only affects 'document_id' scopes)
+- `context_matching_...` param to fine-tune the fuzzy matching mechanism that determines whether some text contexts match each other (only affects 'context' scopes, default values should work most of the time)
+
+**Arguments**:
+
+- `labels`: The labels to evaluate on
+- `documents`: List of List of Document that the first node in the pipeline should get as input per multilabel. Can be used to evaluate a pipeline that consists of a reader without a retriever.
```

Comment:
Depends on the style we prefer. If a word represents an existing class we often times don't use the plural form. Otherwise it would need to be List of Lists of Documents. I would keep it as is.